src/connectivity.jl

```diff
@@ -252,66 +253,63 @@ function strongly_connected_components end
                 v = dfs_stack[end] #end is the most recently added item
                 u = zero_t
                 @inbounds for v_neighbor in outneighbors(g, v)
-                    if index[v_neighbor] == zero_t
-                        # unvisited neighbor found
+                    if is_unvisited(rindex,v_neighbor)
```

Comment:
```suggestion
                    if is_unvisited(rindex, v_neighbor)
```
in general, would suggest to add a space after a comma in arguments lists for better readability and consistency with the other code.